### PR TITLE
Feature/parser

### PR DESCRIPTION
**Pull request by Hediby**
This PR contains two main things:

- [x] Some general repository stuff (`pyproject.toml` , `requirements.txt`) 
- [x] Creation of a base parser that uses the python lib `article-parser` 

TODO:
- [x] Write better tests
- [ ] I noticed that the parser wasn't working with at least two sources (Ouest-France and Frandroid). These should be investigated.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+[tool.setuptools]
+packages = ["src"]
+
+[project]
+name = "veritaserum"
+version = "0.0.1"
+
+[options]
+python_requires = ">=3.6"
```
Comment (nicolaslecrique) [inline pyproject.toml]: i would use a more recent version so we can use recent language features (at least 3.10)

File: src/article.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+@dataclass_json
+@dataclass
+class Article:
+    url: str # only this field is mandatory
+    # title and content are instanciated by the url parser
+    title: Optional[str] = None 
+    content: Optional[str] = None
+    # suggested_title is instanciated by the LLM
+    suggested_title: Optional[str] = None
```
Comment (nicolaslecrique) [inline src/article.py]: i would remove code unused (suggested, meta_data) for now. When we do need it we might need something slightly different anyway

Comment (Hediby) [reply src/article.py]: Done !

File: pyproject.toml (hunk 1 above)
Comment (Hediby) [reply pyproject.toml]: Done

**Review by nicolaslecrique — APPROVED** [review]

**Pull request merged**